### PR TITLE
introducing ck_tile!

### PR DESCRIPTION
**Pull request by carlushuang**
This PR introduces `ck_tile`, next generation of composable kernel.

`ck_tile` is independently from the old ck, located under `/include/ck_tile`. You don't need to include anything from old CK, `ck_tile` has similiar (indeed almost the same) implementations for users to build operators. We will have a transition period to pull everything from old ck into `ck_tile`, stay tuned.

Currently we prepared `fmha` example under `example/ck_tile/01_fmha/`, please check the README under that folder

### CODE REVIEW

File: example/ck_tile/01_fmha/generate.py [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+                    pipelines.append(FmhaFwdPipeline('qr_async', 'row', 't', 'f', 't', 't', bias, lse, mask))
+                    pipelines.append(FmhaFwdPipeline('qr_async', 'row', 't', 't', 't', 't', bias, lse, mask))
+                    pipelines.append(FmhaFwdPipeline('qr_async', 'col', 't', 'f', 't', 't', bias, lse, mask))
+                    pipelines.append(FmhaFwdPipeline('qr_async', 'col', 't', 't', 't', 't', bias, lse, mask))
+        elif dtype in ['fp8', 'bf8']:
+            # no need lse kernels
+            for mask, bias in itertools.product(MASK_MAP.keys(), ["t", "f"]):
+                pipelines.append(FmhaFwdPipeline('qr_fp8', 'col', 'f', 'f', 'f', 'f', bias, 'f', mask))
+        else:
+            assert Fasle
```
Comment (rocking5566) [inline example/ck_tile/01_fmha/generate.py]: fix typo

Comment (carlushuang) [reply example/ck_tile/01_fmha/generate.py]: fixed

**Review by zjing14 — APPROVED** [review]

**Review by zjing14 — DISMISSED** [review]

**Comment by asroy** [comment]
Next-Gen Baby!

**Review by zjing14 — DISMISSED** [review]

**Review by zjing14 — APPROVED** [review]

**Comment by zjing14** [comment]
@ROCm/rocm-documentation Please approve the PR for merging.

**Review by poyenc — APPROVED** [review]

**Review by rocking5566 — APPROVED** [review]

**Comment by carlushuang** [comment]
@ROCm/rocm-documentation Hi, please help approve this PR

File: example/ck_tile/01_fmha/README.md
Comment (randyh62) [inline example/ck_tile/01_fmha/README.md:1]: Currently we didn't do much thing at the epilogue stage but leave the room for future possible support.

This sentence should be removed.

**Review by randyh62 — APPROVED** [review]

**Pull request merged**